### ISSUE TITLE
Feature Request: Have dark mode apply to extension (settings, etc) pages

### ISSUE THREAD

**Issue by srsutherland** [deanoemcke/thegreatsuspender#992]
* Extension v7.1.0
 * Chrome 77

The new UI looks nice, but it's a really blinding white for those of us that have the rest of our browser set to dark mode. It'd be nice if the CSS for extension pages actually respected the "dark mode" setting and had white-text-on-dark-background.

Pages in question:

* chrome-extension://klbibkeccnjlkjkiokjodocebajanakg/updated.html
* chrome-extension://klbibkeccnjlkjkiokjodocebajanakg/options.html
* chrome-extension://klbibkeccnjlkjkiokjodocebajanakg/history.html
* chrome-extension://klbibkeccnjlkjkiokjodocebajanakg/history.html
* chrome-extension://klbibkeccnjlkjkiokjodocebajanakg/shortcuts.html
* chrome-extension://klbibkeccnjlkjkiokjodocebajanakg/about.html

**Comment by angelawyman** [comment]
I second this- the page is so blinding after dark mode-ing other pages.  Would love my browsing experience to be more seamless. Thanks!

**Comment by omfganimal** [comment]
Yes please do this. I thought the settings page was supposed to do that when set to Dark theme, but apparently it's only for paused tabs. This would be great!
